fairnessbench_analysis/explode_results.py

```diff
@@ -5,7 +5,7 @@
 import json 
 
 # loading the performance results
-perf_path = '../results_manually_combined'
+perf_path = '/project/pi_brownsarahm_uri_edu/ayman_uri/fairnessBench/results_final_total'
```

Comment:
let's put these absolute paths in an ignored file, `paths.py` and then add a readme that indicates what should be in that file